core/app/components/StageSelect/StageSelectServer.tsx

```diff
@@ -8,28 +8,12 @@ type Props = {
 	communityId: CommunitiesId;
 	fieldLabel: string;
 	fieldName: string;
-	value?: StagesId;
 };
 
-export async function StageSelectServer({ communityId, fieldLabel, fieldName, value }: Props) {
```

Comment:
I don't think passing the stage's value is actually needed here—the form takes care of that from what I can tell